_dashboards/management/management-index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: Dashboards Management
+nav_order: 110
+has_children: true
+---
+
+# Dashboards Management
+
+Dashboards Management serves as the command center for customizing OpenSearch Dashboards to your needs. 
+
+OpenSearch and OpenSearch Dashboards privileges govern access to individual features. If you do not have the appropriate access, consult your administrator. 
+{: .tip}
+
+The following applications are available in Dashboards Management:
+
+- **Index Patterns:** To access OpenSearch data, you need to create an index pattern so that you can select the data you want to use and define the properties of the fields. An index pattern can point to one or more indexes, data streams, or index aliases. 
```

Comment:
This should probably the #1 priority for missing docs - creating index patterns is an essential first step to doing anything else with your data in OpenSearch Dashboards. The only reason it can be skipped in the Quickstart (https://opensearch.org/docs/2.9/dashboards/quickstart/) is the fact that the index patterns are already created for the sample data sets.

Reply:
I agree, this is honestly the biggest source of confusion for new users and the thing they are most likely to search for.